src/index.ts

```diff
@@ -1 +1,2 @@
 export * from './shared-components';
+export * from './constants';
```

Comment:
Since we just need the theme, is there any objection to just `export { primaryTheme, secondaryTheme } from './constants';`?

Reply:
I think it would be helpful to have the other constants available for consistency but it's not as important as the themes.

I was just thinking we should export everything that's part of the public API so that the available exports don't differ depending on which module system you're using.

Reply:
I'm cool with that. It looks like `dist` pretty much has all of them anyway, we just use relative imports within the library.

***

As an aside, one thing I've wanted to do is survey other major open source libraries and figure out what some "best practices" are that we're not doing. Blueprint, for example, uses the `@blueprint` namespace and has its exports from different paths/packages:

```tsx
import { Icon, Intent } from "@blueprintjs/core";
import { IconNames } from "@blueprintjs/icons";
```

I think *trying* to maintain that for us--with `lib/constants` and `/lib/icons` as the candidates--would be helpful, but I don't think it's a priority with the other stuff for us to get done this cooldown. I don't expect tons of people (anyone, really) to start pulling in the constants directly from `radiance-ui` after making these changes.